assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -42,43 +37,44 @@ interface StandardLayoutProps {
   remedy: string;
   effect: string;
   location: string | null;
-  bannerHeight: number;
   disruptionDiagram?: DisruptionDiagramData;
 }
 
-// For the standard layout, issue font can be medium or large.
-// If remedy is "Seek alternate route", font size is static. Otherwise, it uses the same font size as
-// the issue.
+// Bypassed station alerts can have resizing font based on how many stations are affected
+// Other alerts have static font sizes:
+// - issue font is size large
+// - "Seek alternate route" remedy is medium
+// - "Use shuttle bus" remedy is large
 const StandardLayout: React.ComponentType<StandardLayoutProps> = ({
   issue,
   remedy,
   effect,
   location,
-  bannerHeight,
   disruptionDiagram,
 }) => {
-  const maxTextHeight =
-    SCREEN_HEIGHT -
-    (FOOTER_HEIGHT + BOTTOM_MARGIN + ALERT_CARD_PADDING + bannerHeight);
+  const maxTextHeight = 772
 
   const { ref: contentBlockRef, size: contentTextSize } = useTextResizer({
     sizes: ["medium", "large"],
-    maxHeight: maxTextHeight,
+    // the 32 is padding on the text object
+    maxHeight: maxTextHeight + 32,
     resetDependencies: [issue, remedy],
   });
 
   return (
-    <div className="alert-card__content-block" ref={contentBlockRef}>
-      <StandardIssueSection
-        issue={issue}
-        location={location}
-        contentTextSize={contentTextSize}
-      />
-      <RemedySection
-        effect={effect}
-        remedy={remedy}
-        contentTextSize={contentTextSize}
-      />
+    <div className="alert-card__content-block" >
+      <div ref={contentBlockRef}>
+        <StandardIssueSection
+          issue={issue}
+          location={location}
+          contentTextSize={effect === "station_closure" ? contentTextSize : "large"}
```

Comment:
Similar question as my other about the remedy. Removing this prop would also mean an effect prop would be needed, so not sure if there's much benefit for this one.